### ISSUE TITLE
More detail about the test result in Argoverse2 Leaderboard

### ISSUE THREAD

**Issue by Morphlingxxx** [fudan-zvg/RealMotion#7]
Hi,

Thank you for your great work.

I've successfully reproduced the validation set results, which match those reported in your paper. However, when I submit the model to the test set, the performance falls significantly short of the leaderboard's results.

Could you please provide more details regarding the realmotion method's performance on the test set? Are there additional steps or specific configurations required for the test set that might have been omitted in the documentation? Any insights or clarifications would be greatly.

**Comment by Morphlingxxx** [comment]
Sorry to have another question. In the answer to ”Questions On AV2 Val/Test Scores”, you mentioned the test weights obtained by training with the trainval set. 

Is there no difference in the model itself? 

Is it just the division of the training set that leads to the improvement in performance?

**Comment by NaNaoiSong** [comment]
Apologies for that after the refactoring, we did not resubmit to the leaderboard. Could you share the test performance you obtained to help us identify any issues?

**Comment by Morphlingxxx** [comment]
> Apologies for that after the refactoring, we did not resubmit to the leaderboard. Could you share the test performance you obtained to help us identify any issues?

I apologize for the late response. The results of our resubmission to the leaderboard are as follows:

[{'minFDE (K=6)': 1.2948, 
'minFDE (K=1)': 4.1207, 
'minADE (K=6)': 0.6729, 
'minADE (K=1)': 1.6539, 
'MR (K=6)': 0.1547,
 'MR (K=1)': 0.5847, 
'brier-minFDE (K=6)': 1.9328}]

**Comment by Morphlingxxx** [comment]
The validation results are as follows:
│            MR             │    0.15928137302398682    │
│         b-minFDE6         │     1.935653805732727     │
│          minADE1          │    1.6467903852462769     │
│          minADE6          │    0.6694254279136658     │
│          minFDE1          │     4.100405693054199     │
│          minFDE6          │    1.3034820556640625     │
│    val/step0_reg_loss     │    0.1864209920167923     │
│    val/step1_reg_loss     │    0.18882419168949127    │
│    val/step2_reg_loss     │    0.19656476378440857

**Comment by Morphlingxxx** [comment]
> Apologies for that after the refactoring, we did not resubmit to the leaderboard. Could you share the test performance you obtained to help us identify any issues?

Hello,

**Comment by NaNaoiSong** [comment]
Apologies for the late response. We are sorry that we couldn't find the detailed configurations for submission, but we primarily try increasing epochs, training on the trainval set, and adding more Transformer layers. We will provide a submission configuration later.

**Comment by Morphlingxxx** [comment]
> Apologies for the late response. We are sorry that we couldn't find the detailed configurations for submission, but we primarily try increasing epochs, training on the trainval set, and adding more Transformer layers. We will provide a submission configuration later.

Appreciate the info on your approach, and we’re excited to see the updated submission config when it’s ready.
